jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessorTest.java

```diff
@@ -842,6 +842,18 @@ public void testGetJavaContainerBuilderWithBaseImage_incompatibleJava11JettyBase
     }
   }
 
+  // https://github.com/GoogleContainerTools/jib/issues/1995
+  @Test
+  public void testGetJavaContainerBuilderWithBaseImage_java12BaseImage()
```

Comment:
Should we add a test that fails if java version is `>11` and no image is set?

Reply:
Sure